nservicebus/upgrades/externalize-encryption.md

```diff
@@ -36,4 +35,23 @@ snippet: SplitDefiningEncryptedPropertiesAs
 
 ## Custom encryption Service
 
-snippet: SplitEncryptionFromIEncryptionService
\ No newline at end of file
+snippet: SplitEncryptionFromIEncryptionService
+
+
+## Compatibility
+
+The NServiceBus.Encryption.MessageProperty package is partially compatible with endpoints using NServiceBus package's encryption functionality:
+
+
+### Sending and receiving using NServiceBus.Encryption.MessageProperty
```

Comment:
I'd consider something like "Encrypting and decrypting with NServiceBus.Encryption.MessageProperty" for the title since the section doesn't have anything to do with sending/receiving. Maybe even "Supported types when encrypting and decrypting with NServiceBus.Encryption.MessageProperty"